src/Adobe/Provider.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace SocialiteProviders\Adobe;
+
+use GuzzleHttp\Exception\GuzzleException;
+use SocialiteProviders\Manager\Contracts\OAuth2\ProviderInterface;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider implements ProviderInterface
+{
+    public const IDENTIFIER = 'ADOBE';
+    public const BASE_URL = 'https://ims-na1.adobelogin.com/ims';
+
+    protected array $scopes = ['openid', 'email', 'profile'];
+
+    protected function getAuthUrl($state): string
+    {
+        return $this->buildAuthUrlFromBase(self::BASE_URL.'/authorize', $state);
+    }
+
+    protected function getTokenUrl(): string
+    {
+        return self::BASE_URL.'/token';
+    }
+
+    protected function getTokenFields($code): array
+    {
+        return [
+            ...parent::getTokenFields(),
```

Comment:
Is this supported in 7.4? Just checking :P

Reply:
Just checked it out, seems to be supported since PHP 8.0. Will refactor it to support 7.4 syntax!